code/src/index.css

```diff
@@ -1,13 +1,170 @@
 body {
   margin: 0;
+  padding: 0;
+  width: 100%;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(24 24 24);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main {
+  padding-bottom: 70px;
+}
+
+.albums {
+  display: grid;
+  grid-template-columns: repeat(4, minmax(50px, 1fr));
+  grid-template-rows: repeat(auto, minmax(50px, 1fr));
+  padding: 20px;
+  row-gap: 35px;
+  column-gap: 20px;
+}
+
+.master-wrapper {
+  display:inline-block;
+  padding: 0px;
+}
+
+/* ALBUM COVER IMAGE*/
+.album-cover-overlay {
+  position: relative;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.album-cover {
+  width: 100%;
+  object-fit: cover;
+}
+
+.album-name-link {
+  display: inline-block;
+}
+
+/* ICONS*/
```

Comment:
Excellent way of working with the icons position and their hover effects. Robust!